src/runtime_src/core/pcie/tools/xbmgmt/cmd_flash.cpp

```diff
@@ -660,6 +665,78 @@ static int update(int argc, char *argv[])
     if (shell.empty() && !id.empty())
         return -EINVAL;
 
+    if(!xclbin.empty())
```

Comment:
for consistency with other conditionals in this file, please add space between 'if' and '('